specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2021-02-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1727 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-02-01",
+    "title": "AppServiceCertificateOrders API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CertificateRegistration/certificateOrders": {
+      "get": {
+        "tags": [
+          "AppServiceCertificateOrders"
+        ],
+        "summary": "List all certificate orders in a subscription.",
+        "description": "Description for List all certificate orders in a subscription.",
+        "operationId": "AppServiceCertificateOrders_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AppServiceCertificateOrderCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2021-02-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CertificateRegistration/validateCertificateRegistrationInformation": {
+      "post": {
```

Comment:
>post [](http://example.com/codeflow?start=7&length=4)

How does this operation indicate a validation failure?

Reply:
Unfortunately, the current implementation only throws an exception.